modules/helm-installing-a-helm-chart-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="installing-a-helm-chart-on-an-openshift-cluster.adoc_{context}"]
+
+= Installing a Helm chart on an OpenShift cluster
+
+.Prerequisites:
+* You have a running {prod-title} cluster and you have logged into it.
```

Comment:
Several instances of `{prod-title}` in this PR need to be `{prod-title}`. Try to remember to look at the rendered version of the docs to see if anything like this isn't being replaced as expected.